examples/src/main/scala/example/tapir/ExampleApp.scala

```diff
@@ -33,16 +34,18 @@ object ExampleApp extends CatsApp {
   val booksListingEndpoint: ServerEndpoint[(Option[Int], Option[Int]), Nothing, List[Book], Any, UIO] =
     booksListing.serverLogic[UIO] { case (year, limit) => bookListingLogic(year, limit).map(Right(_)) }
 
-  val graphql2: GraphQL[Any]                                                                          =
+  val graphql2: GraphQL[Any] =
     addBookEndpoint.toGraphQL |+| deleteBookEndpoint.toGraphQL |+| booksListingEndpoint.toGraphQL
 
+  type MyTask[A] = RIO[Clock with Blocking, A]
+
   override def run(args: List[String]): ZIO[ZEnv, Nothing, ExitCode] =
     (for {
       interpreter <- graphql.interpreter
-      _           <- BlazeServerBuilder[Task]
+      _           <- BlazeServerBuilder[MyTask]
                        .bindHttp(8088, "localhost")
                        .withHttpApp(
-                         Router[Task](
+                         Router[MyTask](
                            "/api/graphql" -> CORS.policy(Http4sAdapter.makeHttpService(interpreter)),
```

Comment:
So do we still use `Http4sAdapter` or does this change to a generic `TapirAdapter`?

Reply:
While technically users could directly use `TapirAdapter`, I was thinking to keep `Http4sAdapter` and others to make it even easier.
For http4s and zio-http it's a 2-liner so the benefit is not huge, but for akka-http (and play, I guess) it requires some code to transform between their types (`Future`, `Flow`) and ZIO types.